ocaml/idl/api_messages.ml

```diff
@@ -96,6 +96,7 @@ let v6_license_server_version_obsolete = addMessage "LICENSE_SERVER_VERSION_OBSO
 let pvs_proxy_no_cache_sr_available = addMessage "PVS_PROXY_NO_CACHE_SR_AVAILABLE" 3L (* No cache storage available for pvs site on the host *)
 let pvs_proxy_setup_failed = addMessage "PVS_PROXY_SETUP_FAILED" 3L (* Setting up pvs proxy rules or pvs proxy daemon initialisation failed internally *)
 let pvs_proxy_no_server_available = addMessage "PVS_PROXY_NO_SERVER_AVAILABLE" 3L (* No pvs server available for pvs site on host *)
+let pvs_proxy_cache_invalid_size = addMessage "PVS_PROXY_CACHE_INVALID_SIZE" 3L (* Cache storage VDI size exceeds available control domain memory *)
```

Comment:
Would this be good?:
`let pvs_proxy_cache_too_big_for_sr = addMessage "PVS_PROXY_CACHE_TOO_BIG_FOR_SR" 3L (* PVS cache size exceeds available space on SR *)`

I feel like it's not as specific as `PVS_PROXY_CACHE_SIZE_EXCEEDS_SR_AVAILABLE_SPACE` or `PVS_PROXY_CACHE_SIZE_EXCEEDS_SR_FREE_SPACE`, but these seem too long if `PVS_PROXY_CACHE_TOO_BIG_FOR_SR` is enough?